applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.cpp

```diff
@@ -146,14 +146,24 @@ void UPwBaseElement::Initialize(const ProcessInfo& rCurrentProcessInfo)
             std::fill(r_stress_vector.begin(), r_stress_vector.end(), 0.0);
         }
     }
+    std::vector<Vector> strain_vectors;
+    strain_vectors.resize(number_of_integration_points);
+    for (auto& r_strain_vector : strain_vectors) {
+        r_strain_vector.resize(GetStressStatePolicy().GetVoigtSize());
+        std::fill(r_strain_vector.begin(), r_strain_vector.end(), 0.0);
+    }
```

Comment:
In theory, I think this could be shortened to: 
```suggestion
    std::vector<Vector> strain_vectors(number_of_integration_points, ZeroVector(GetStressStatePolicy().GetVoigtSize());
```